readme.md

```diff
@@ -125,6 +129,10 @@ as you will always be prompted to create secrets if necessary. The following tab
 
 For known issues with `npx setup-ci`, please check [Troubleshooting](docs/troubleshooting.md).
 
+## 📊 Metrics
+
+By default, `npx setup-ci` collects anonymous usage data that helps us improve the tool. You can disable it by using the `--skip-telemetry` option. Collected data consists of timestamp, tool version, selected options, whether the script has been run for the first time and whether it ended with an unexpected error.
```

Comment:
I would add that error doesn't contain stacktrace that could potentially have some user details.

Reply:
added in 4a65a27c513cca5bdfea854e4c675f980753beb4